backend/routers/user.py

```
@@ -4,9 +4,17 @@
 import os
 from datetime import datetime, timezone
 import uuid
-from typing import Dict, Any
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)